### ISSUE TITLE
Existing connections are not showing up on Strongman Portal

### ISSUE THREAD

**Issue by zubinaryaEyenuk** [strongswan/strongMan#152]
Connections which already existed before Strongman installation are not showing up on the portal.
swanctl --list-conns however displays the connections

**Comment by tobiasbrunner** [comment]
Yes, strongMan does not show such connections. It only shows connections it configures itself (it has its own database with configs independent of swanctl.conf or other connections configured via vici).

**Comment by Spider84** [comment]
Can other configurations be showed?
